src/helpers/helpers.ts

```diff
@@ -0,0 +1,73 @@
+import type { DIDDocument } from 'did-resolver'
+import type { MethodSpecificIdAlgo, CheqdNetwork } from '@cheqd/sdk'
+
+import { VerificationMethods, createVerificationKeys, createDidVerificationMethod, createDidPayload } from '@cheqd/sdk'
+import { rawSecp256k1PubkeyToRawAddress } from '@cosmjs/amino'
+import { toBech32 } from '@cosmjs/encoding'
+import { publicKeyConvert } from 'secp256k1'
+import { fromString } from 'uint8arrays'
+
+import { SpecValidationResult } from '../types/types'
+
+export function validateSpecCompliantPayload(didDocument: DIDDocument): SpecValidationResult {
```

Comment:
Do we have a validation already in "@cheqd/sdk" or in "@cheqd/did-provider-cheqd" packages? Otherwise I think it would be better to keep it somewhere in such packages